11-taxonomic-information.Rmd

```diff
@@ -158,6 +158,72 @@ assay(altExp(se, "Family"), "counts")[1:5,1:7]
 `altExpNames` now consists of `Family` level data. This can be extended to use 
 any level present in `r mia::taxonomyRanks(se)`.   
 
+## Data transformation
+
+Data transformation is very common procedure in microbiome analysis. 
+In transformation, each data point is replaced with transformed value that is 
+calculated by applying transformation formula to the data point. Transformation 
+can be used, for example, to normalize skewed data, or to reduce weight of bigger 
+values compared to smaller values. 
+
+In mia package, transformations are applied to abundance data. Transformed 
+abundance table is stored back to 'assays'. mia includes transformation 
+functions for sample-wise or column-wise transformation ('transformSamples()'), 
+and for feature-wise or row-wise transformation ('transformFeatures()'). 
+
+'transformSamples' includes following transformation methods:
+
+-   centered log ratio transformation (clr)
+-   hellinger (hellinger)
+-   log10 (log10)
+-   presence/absence (pa)
+-   rank (rank)
+-   relative (relabundance)
+
+'transformFeatures' includes following transformation methods:
+
+-   log10 (log10)
+-   presence/absence (pa)
+-   Z-standardization (z)
```

Comment:
To keep this short and to avoid replication and possible outdating of this text, I would suggest just referring to mia documentation on these functions (found via [mia homepage](https://microbiome.github.io/mia/reference/index.html)) and stating "For a complete list of available transformations, see the function help."

Reply:
I agree, removed